components/camera/camera.go

```diff
@@ -171,6 +160,49 @@ func DecodeImageFromCamera(ctx context.Context, mimeType string, extra map[strin
 	return img, nil
 }
 
+// GetImageFromGetImages is a utility function to quickly implement GetImage from an already-implemented GetImages method.
+// It returns a byte slice and ImageMetadata, which is the same response signature as the Image method.
+// We use the mimeType arg to specify how to encode the bytes returned from GetImages, returning the first image in the response slice.
+func GetImageFromGetImages(ctx context.Context, mimeType string, cam Camera) ([]byte, ImageMetadata, error) {
```

Comment:
I recommend taking the sensor name as a string pointer and if the pointer is not null, only returning the image that matches the sensor name. 

If the pointer is nil then you can arbitrarily pick the first one.

Reply:
what is the behavior if mimeType is empty? Can we please have a test for that?

Reply:
Should error because it hits the unknown default case in the encode helper. Added test

Reply:
>I recommend taking the sensor name as a string pointer and if the pointer is not null, only returning the image that matches the sensor name.

Added